docs/3.0/deploy/index.mdx

```diff
@@ -14,19 +14,200 @@ Attributes of a deployment include:
 - **Schedule** or **Trigger**: optional schedule or triggering rules for this deployment
 - **Tags**: optional text labels for organizing deployments
 
-Some of the most common reasons to use an orchestration tool like Prefect are for
+Some of the most common reasons to use an orchestration tool such as Prefect are for
 [scheduling](/3.0/automate/add-schedules/) and [event-based triggering](/3.0/automate/events/automations-triggers/).
-As opposed to manually triggering and managing flow runs, deploying a flow exposes an API and
+In addition to manually triggering and managing flow runs, deploying a flow exposes an API and
 UI that allow you to:
 
 - trigger new runs, [cancel active runs](/3.0/develop/write-flows/#cancel-a-flow-run),
 pause scheduled runs, customize parameters, and more
-- remotely configure schedules and automation rules for your deployments
-- dynamically provision infrastructure using [workers](/3.0/deploy/infrastructure-examples/docker)
+- remotely configure [schedules](/3.0/automate/add-schedules) and [automation rules](/3.0/automate/events/automations-triggers) for deployments
+- dynamically provision infrastructure with [work pools](/3.0/deploy/infrastructure-concepts/work-pools)
 
-## Deployment schema
+## Create a deployment 
+
+### Create a deployment in Python with a local `serve` method
+
+This is the simplest way to get started with deployments:
+
+```python
+from prefect import flow
+
+
+@flow
+def my_flow():
+    print("Hello, Prefect!")
+
+
+if __name__ == "__main__":
+    my_flow.serve(name="my-first-deployment", cron="* * * * *")
+```
+
+The `serve` method creates a deployment from your flow and immediately begins listening for scheduled runs to execute. 
+It also create a schedule so that the flow will run every minute of every day.
+
+For slightly more configuration you can create a deployment that uses a work pool.
+Reasons to create a work-pool based deployment include:
+
+- Wanting to run your flow dynamically on remote infrastructure
+- Needing more control over the execution environment
+- Creating a template for other deployments
+
+Work pools are popular with data platform teams because they allow you to manage infrastructure configuration across an organization. 
+
+### Create a work pool-based deployment
+
+Prefect offers two primary options for creating deployments that use a work pool: a Python method and a YAML file.
+
+### Create a deployment with a Python `flow.deploy` method
```

Comment:
I suggest this name to mirror the serve section above, but it could be `flow.deploy` instead
```suggestion
### Create a deployment with `deploy`
```